src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double base;
+    private final double lateralSide;
+
+    IsoscelesTrapezoid(String color, double base, double lateralSide) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables and pass them on for better code readability. Although your constructor is not using expressions directly, ensure that the values passed to it are not expressions either.

Reply:
I dont pass any expressions in the constructor